StockAnalysis.Tests/DiffTests/DiffStoreTests/PdfDiffStoreTests.cs

```diff
@@ -0,0 +1,35 @@
+using StockAnalysis.Diff.Store;
+using StockAnalysisTests.Utility;
+
+namespace StockAnalysisTests.DiffTests.DiffStoreTests;
+
+public class PdfDiffStoreTests
+{
+
+    private static void CheckValidPdfFormat(string path)
+    {
+        using var document = new Aspose.Pdf.Document(path);
+        Assert.That(document.Pages, Has.Count.GreaterThanOrEqualTo(3));
+    }
+
+    [Test]
+    public async Task StoreDiff_WhenCalledRight_ShouldCreateFile()
```

Comment:
Tady bych se zkusil zamyslet, jestli by nesly testy pro vsechny formaty sjednotit a parametrizovat. Ale v rozsahu naseho predmetu je to i takhle ok.